bazel_tools/build_environment.bzl

```
@@ -12,11 +21,17 @@ def _impl(ctx):
     # Generates a simple Bazel file that just sets a bunch of Bazel variables,
     # so they can be used in our main Bazel BUILD files.
     semver = ctx.os.environ.get("DAML_SDK_RELEASE_VERSION", default = "0.0.0")
-    if semver.find("-snapshot.") > 0:
-        ghc = semver[:-9].replace("-snapshot.", ".")
+    if semver.find("-") > 0:
+        ghc_list = []
+        for segment in semver.replace("-", ".").split("."):
+            if is_number(segment):
```

```suggestion
            if segment.isdigit():
```
and remove all the stuff above.

---

Oh, missed that.